pythonforandroid/bootstraps/sdl2/build/src/patches/SDLActivity.java.patch

```diff
@@ -16,19 +25,19 @@
          // Load shared libraries
          String errorMsgBrokenLib = "";
          try {
-@@ -639,7 +648,7 @@ public class SDLActivity extends Activity implements View.OnSystemUiVisibilityCh
+@@ -639,7 +650,7 @@
      Handler commandHandler = new SDLCommandHandler();
- 
+
      // Send a message from the SDLMain thread
 -    boolean sendCommand(int command, Object data) {
 +    protected boolean sendCommand(int command, Object data) {
```

Comment:
Oh god it's so hard to read a diff of a diff, I initially tought you changed that.
You actually just removed some trailing whitespace so GitHub is showing the context which contains the previous diff, that's so confusing to read :exploding_head: but that's fine I'll manage